core/src/main/java/org/apache/calcite/sql/SqlWindowTableFunction.java

```diff
@@ -98,8 +173,8 @@ private static RelDataType inferRowType(SqlOperatorBinding opBinding) {
     return typeFactory.builder()
         .kind(inputRowType.getStructKind())
         .addAll(inputRowType.getFieldList())
-        .add("window_start", timestampType)
-        .add("window_end", timestampType)
+        .add("wstart", timestampType)
+        .add("wend", timestampType)
```

Comment:
Note that I asked one of the authors of the "one sql to rule them all" paper. The only reason that they use wstart and wend was because of the layout for a paper. 

I also believe in a JIRA or an email (sorry I cannot recall which one it is) we agreed to switch to window_start and window_end.

So I will suggest we keep window_start and window_end, which can be understood easier.

Reply:
Well, I didn’t notice that, can you share the discussion?

Reply:
I will spend some time to search for that JIRA.

Reply:
ok. I might or might not discussed this in Calcite (because I cannot find a JIRA that about it), but

1. This is the acknowledge from Kenneth Knowles, one of the paper authors: https://docs.google.com/document/d/138uA7VTpbF84CFrd--cz3YVe0-AQ9ALnsavaSE2JeE4/edit?disco=AAAAEAjWvxg

2. I started to use window_start and window_end in https://issues.apache.org/jira/browse/CALCITE-3272. Julian, as another paper author, didn't against it I think.

Reply:
Thanks for the share, i have reverted the rename back.

Reply:
THank you!